akka-stream-tests/src/test/scala/akka/stream/scaladsl/SourceWithContextSpec.scala

```diff
@@ -93,8 +91,8 @@ class SourceWithContextSpec extends StreamSpec {
           List(1, 2, 3, 4).map(i ⇒ s"$str-$i")
         }
         .grouped(2)
-        .asSource
-        .runWith(TestSink.probe[(Seq[String], Seq[Long])])
+        .toMat(TestSink.probe[(Seq[String], Seq[Long])])(Keep.right)
+        .run
```

Comment:
Should we leave one test actually using `.asSource` directly?

Reply:
^